changelog/unreleased/3116

```diff
@@ -0,0 +1,6 @@
+Bugfix: Changed share icons to people icons
```

Comment:
Let's maybe use "to collaborators icons"?

Reply:
ok